data/ka'het/ka'het outfits.txt

```diff
@@ -223,3 +223,87 @@ outfit "Ka'het Reserve Accumulator"
 	"outfit space" -3
 	"energy capacity" 600
 	description "This tiny battery was developed for use by the Ka'mar drones as an emergency energy storage, in case their generators can't quite suffice for the high power demand of most Builder systems."
+
+# Engines:
+
+outfit "Maeri Engine Nacelles"
+	category "Engines"
+	"cost" 928000
+	thumbnail "outfit/ka'het maeri engine"
+	"mass" 52
+	"outfit space" -76
+	"engine capacity" -59
+	"energy consumption" 5.01
+	"heat generation" 1.1
+	"thrust" 37.1
+	"thrusting heat" 2.8
+	"turn" 931.4
+	"turning heat" 1.3
+	"flare sprite" "effect/ka'het flare/large"
+		"frame rate" 30
+	"steering flare sprite" "effect/ka'het flare/large"
+		"frame rate" 30
+	"flare sound" "plasma medium"
+	description "The Ka'het engines are peculiar in that they need to be continously charged with large amounts of energy: while this means that less power remains available for other subsystems, it is somewhat more efficient than most types of propulsions at doing so, and generates only a tiny amount of heat."
```

Comment:
> at doing so

At doing what? You might consider clarifying here or rewording in a way that makes it more clear what these engines are more efficient at.

Reply:
I imagine it refers to charging, but yes, this needs to be reworded/clarified.

Reply:
reworded